### ISSUE TITLE
Feture request - client should be able to assign uploaded files to a category

### ISSUE THREAD

**Issue by barna119** [ignacionelson/ProjectSend#392]
Client should be able to assign uploaded files to a category.

**Comment by grdpeter** [comment]
Yes, Client to be able to assign category to each uploaded file or they can bulk-assigned it later 
that will be very delicious implantation...
thank for taking it into consideration 
signed @serverbroad

**Comment by zackrspv** [comment]
Please consider adding this as a feature!

**Comment by Norec39** [comment]
Users should be able to manage their own categories themselves

**Comment by tshives26** [comment]
+1 on this, looks like its been requested for a few years. Really liking the product so far but this is a necessity for my needs. If this could be implemented, that would be greatly appreciated!

**Comment by ignacionelson** [comment]
This issue is being closed as part of a bulk cleanup. ProjectSend has undergone significant changes since this was reported (including r1720 and r1994), and many older issues no longer apply to the current codebase.

If you are still experiencing this problem on the latest version, please open a new issue with updated details and we'll be happy to look into it. Thank you for your contribution!
